client/blocks/plugins-scheduled-updates-multisite/schedule-list.tsx

```diff
@@ -63,16 +65,23 @@ export const ScheduleList = ( props: Props ) => {
 		closeRemoveConfirm();
 	};
 
-	const filteredSchedules = schedules?.filter( ( schedule ) => {
-		if ( ! search || ! search.length ) {
-			return true;
-		}
+	const filteredSchedules = schedules
+		?.map( ( schedule ) => {
+			if ( ! searchTerm || ! searchTerm.length ) {
+				return schedule;
+			}
+
+			const filteredSites = schedule.sites.filter( ( site ) =>
+				site.title.toLowerCase().includes( searchTerm.toLowerCase() )
```

Comment:
I think we can move `searchTerm.toLowerCase()` outside of the filter, so the string is not transformed for all the sites.

Reply:
What about stating either with the title and URL?